proto.com.autodesk.fbx/Editor/Scripts/PostProcessAddToBuild.cs

```
@@ -0,0 +1,119 @@
+    public class PostProcessAddToBuild
+    {
+        private const string fbxsdkNativePlugin = "UnityFbxSdkNative";
+        private const string fbxsdkNativePluginPath = "Packages/com.autodesk.fbx/Editor/Plugins/x86_64";
+        
+        private const string fbxsdkNativePluginExtWin = ".dll";
+        private const string fbxsdkNativePluginExtOSX = ".bundle";
+        private const string fbxsdkNativePluginExtLinux = ".so";
+
+        private const string buildPluginPathWin = "{0}_Data/Plugins";
+        private const string buildPluginPathOSX = "{0}.app/Contents/Plugins";
+        private const string buildPluginPathLinux = "{0}_Data/Plugins";
+
+        [PostProcessBuild(1)]
+        public static void OnPostprocessBuild(BuildTarget target, string pathToBuiltProject)
+        {
+            var buildPathWithoutExt = Path.ChangeExtension(pathToBuiltProject, null);
+
+            string destPath = null;
+            string sourcePath = null;
+            string sourcePathExt = null;
+            switch (target)
+            {
+                case BuildTarget.StandaloneWindows64:
+                    destPath = string.Format(buildPluginPathWin, buildPathWithoutExt);
+                    sourcePathExt = fbxsdkNativePluginExtWin;
+                    break;
+                case BuildTarget.StandaloneOSX:
+                    destPath = string.Format(buildPluginPathOSX, buildPathWithoutExt);
+                    // Since the bundle is technically a folder and not a file, need to copy the contents of the bundle
+                    destPath = Path.Combine(destPath, fbxsdkNativePlugin + fbxsdkNativePluginExtOSX);
+                    sourcePathExt = fbxsdkNativePluginExtOSX;
+                    break;
+                case BuildTarget.StandaloneLinux64:
+                    destPath = string.Format(buildPluginPathLinux, buildPathWithoutExt);
+                    sourcePathExt = fbxsdkNativePluginExtLinux;
+                    break;
+            }
```

Should you raise an exception by default on other targets?